model/src/main/java/uk/gov/hmcts/payment/api/service/UserAwareDelegatingCreditAccountPaymentService.java

```diff
@@ -122,6 +123,16 @@ public List<PaymentFeeLink> search(Date startDate, Date endDate) {
         return paymentFeeLinks;
     }
 
+    @Override
+    @Transactional
+    public void deleteByPaymentReference(String paymentReference) {
+        long records = paymentRespository.deleteByReference(paymentReference);
+        LOG.info("Number of deleted records are: {}", records);
```

Comment:
Address all the respective coverage warnings...

Reply:
Updated code